website/docs/10_going-deeper/azure-scale.mdx

```diff
@@ -0,0 +1,42 @@
+import DocCardList from '@theme/DocCardList'
+
+# Scaling Booster Azure Functions
+
+Booster Azure Provider relies on CosmosDB [change feed processor](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/change-feed-processor) to consume new events.
+In CosmosDB, the partition keys are distributed in ranges, where each range represents a physical partition. [Unlike logical partitions,
+physical partitions are an internal implementation of the system and Azure Cosmos DB entirely manages physical partitions](https://learn.microsoft.com/en-us/azure/cosmos-db/partitioning-overview#physical-partitions)
+
+With Booster EventStream functionality, we could define the number of physical partitions our events are split and create instances for each partition.
+
+To enable EventStream, set the `EventStreamConfiguration` in the configuration object:
+
+> **Warning**: Currently, only available for Azure provider.
+
+```typescript
+  config.eventStreamConfiguration = {
+    enabled: true,
+    parameters: {
+      streamTopic: 'test',
+      partitionCount: 3,
+      messageRetention: 1,
+    },
+  }
+```
+
+## Parameters
+* StreamTopic: Define the internal topic name Booster will use.
+* PartitionCount: Number of Event Hub partitions. The number of functions app consumer instances will match the partition count
```

Comment:
I saw a comment in the code saying that changing this will make the resource to be recreated. I would be ideal to include this as a warning in these docs, along with the effects this "recreation" could cause (are events lost?) so that uses can decide accordingly